.github/workflows/release-operator.yaml

```diff
@@ -0,0 +1,35 @@
+name: Release Operator
+
+on:
+  pull_request:
+    types:
+    - closed
+    branches:
+      - 'release-v[0-9]+.[0-9]+'
+  workflow_dispatch: {}
+
+jobs:
+  if_merged:
+    if: github.event.pull_request.merged == true
+    name: Release operator
+    runs-on: ubuntu-latest
+    steps:
+      # TODO: not sure if the `Install gettext-base`is still required
```

Comment:
why it was required before?

Reply:
A bit of coping and pasting going on the workflow was copied from the existing release workflow. And there I could not work out why it was needed. I didn't mean to go back to it, and see what happens when it is removed.